client/src/main/scala/lsug/ui/event/Speaker.scala

```diff
@@ -58,6 +58,9 @@ object Speaker {
       }
       .getOrElse(ProfilePicture(profile.some): TagMod)
 
+  private def pronoun(profile: P.Speaker.Profile) =
+    profile.pronoun.map { p => <.h4(s"${p.show}") }
```

Comment:
A heading implies a semantic section of text.  This would be better placed in a paragraph, or a span with appropriate css styles:
```html
<p>Referred to as <strong>he/him</strong></p>
```
It should’t be a `h4` as this causes problems for screen readers. A screen reader may read the bio as being under the he/him heading.